src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+
+        String[] matrixMachine = {truck.goWork(),bulldozer.goWork(),excavator.goWork(),
+                truck.stopWork(), bulldozer.stopWork(), excavator.stopWork()};
```

Comment:
Using a String array to store the messages from `goWork()` and `stopWork()` methods is not necessary. It would be more efficient to call these methods directly inside the loop without storing the return values. This would also make the code more readable and maintainable.